club/templates/single.html

```diff
@@ -1,28 +1,95 @@
-<!-- wp:template-part {"slug":"header","tagName":"header"} /-->
+<!-- wp:template-part {"slug":"header","theme":"club"} /-->
```

Comment:
I think we should keep the `tagName` on the header, otherwise the container will default to using a `div`.